lib/irc.js

```diff
@@ -752,6 +760,10 @@ Client.prototype._connectionHandler = function() {
     if (this.opt.webirc.ip && this.opt.webirc.pass && this.opt.webirc.host) {
         this.send('WEBIRC', this.opt.webirc.pass, this.opt.userName, this.opt.webirc.host, this.opt.webirc.ip);
     }
+    
+    if (this.opt.extendedJoin) {
```

Comment:
Can you add a link to https://ircv3.net/specs/extensions/extended-join-3.1.html here?